src/tracktour/_io_util.py

```diff
@@ -77,11 +81,17 @@ def get_im_info(centers, labels, im_arr):
 def get_medoid(prop):
     region = prop.image
     region_skeleton = skeletonize(region).astype(bool)
-    if np.sum(region_skeleton) == 1:
+    skeleton_sum = np.sum(region_skeleton)
+    if skeleton_sum == 1:
+        logger.info(
+            f"Region skeleton for {prop.label} is a single pixel, using it as medoid."
+        )
         medoid_offset = np.unravel_index(
             np.argmax(region_skeleton), region_skeleton.shape
         )
     else:
+        if skeleton_sum == 0:
+            logger.warning(f"Region skeleton for {prop.label} is empty.")
```

Comment:
is warning really the right call here? I think if it's 0 central_pixel will *definitely* fail? Or am I wrong? Or is this just so you can "trap" the bug without killing it? 😂

Reply:
mm yes so, it **would** fail after this, but yeah I just wanted this here because if we do see this message in the logfile before the error, we know this is specifically this issue. Whereas if we see the stacktrace and not this message, we know it's gone awry.

I haven't actually gone to implement a fix here cause I wanted to specifically identify the edge case that's failing, and **then** make a release with the bugfix. Rather than changing stuff and it potentially still failing. Thoughts?